Changelog.md

```diff
@@ -3,7 +3,7 @@
 Language Features:
  * Introduce global ``block.blobbasefee`` for retrieving the blob base fee of the current block.
  * Yul: Introduce builtin ``blobbasefee()`` for retrieving the blob base fee of the current block.
-
+ * Yul: Introduce builtin ``blobhash()`` for retrieving versioned hashes of blobs associated with the transaction.
```

Comment:
Oh, one more thing, let's finally stop messing with the spacing here :)

```suggestion
 * Yul: Introduce builtin ``blobhash()`` for retrieving versioned hashes of blobs associated with the transaction.
 
```

I fixed it earlier in all those PRs but it keeps reappearing again and again for some reason. Don't worry about the wrong spacing of the `Compiler Features:` section. We'll fix it when we sort the changelog. In PRs just add an entry for your PR and don't touch other lines.